Lib/idlelib/NEWS.txt

```diff
@@ -3,6 +3,10 @@ Released on 2019-10-20?
 ======================================
 
 
+bpo-38401: Shell restart lines now fill the window width and always
```

Comment:
I find the second sentence hard to grok without more context, and the part about a trailing space overly detailed and technical. How about:

"Shell restart lines now fill the window width, always start with '=', and avoid wrapping unnecessarily. The line will still wrap if the included file name is long relative to the width."

Reply:
Agreed.  I already reworded in the prototype commit message.  Your version is nice.

Reply:
Both news items replaced with your improvement.  Thank you for the help polishing the PR.